.github/workflows/unit_tests.yml

```diff
@@ -12,26 +12,20 @@ jobs:
     name: Run check, test and lints
     runs-on: ubuntu-latest
     env:
-      RUST_TOOLCHAIN_VERSION: nightly-2021-10-24
       CARGO_INCREMENTAL: 0
     steps:
       - name: Checkout Source code
         uses: actions/checkout@v2
 
-      - name: Print Toolchain Version
-        run: echo running with "$RUST_TOOLCHAIN_VERSION"
-
       - name: Install Rust Toolchain
         uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: ${{ env.RUST_TOOLCHAIN_VERSION }}
-          override: true
-          components: rustfmt, clippy
 
-      - name: Install WASM target
-        run: rustup target add wasm32-unknown-unknown --toolchain "$RUST_TOOLCHAIN_VERSION"
+      - name: Install clippy and fmt
```

Comment:
Idea to consider: we could move this and python linting to a seperate workflow. It's a major PITA to find out python linting is off having spent 30 minutes waiting for unit tests to finish

Reply:
make a jira ticket pls?